src/modules/tasksClass.js

```diff
@@ -0,0 +1,88 @@
+let checkboxes;
+let descriptions;
```

Comment:
The checkboxes and descriptions variables are declared at the beginning of the file, but are only assigned values later in the code. It would be clearer to declare these variables where they are assigned values, like this:

```
updateTasks = () => {
  this.getFromLocalStorage();
  const temp = this.tasks;
  const checkboxes = document.querySelectorAll('input[type=checkbox]'); // declare checkboxes variable here
  const descriptions = document.querySelectorAll('.task-description'); // declare descriptions variable here

```